src/styles/_landing.scss

```diff
@@ -214,37 +229,71 @@
         color: $gold-leaf;
         user-select: none;
         cursor: default;
-
       }
-
     }
   }
 
   #tell-me-more {
     position: relative;
-    background: url('/assets/img/landing/background-chair.jpg');
     background-size: cover;
     background-position-x: 50%;
     z-index: 0;
-    > :after {
-      content: ('');
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      background: url('/assets/img/landing/background-chair-overlay.png');
-      background-size: cover;
-      z-index: 1;
-      transform: translate(-50%, -50%);
-      top: 49%;
-      left: 50%;
+
+    // Reasoning for responsive:
+    // The element containing the background image will be the following sizes per screen width.
```

Comment:
Deciding responsive breakpoints can be a bit of a brainfuck. Always good to leave comments for next time we need to tweak this area. Comments are removed on compile.